### PR TITLE
Add info support

### PR DESCRIPTION
**Pull request by Shougo**
I have added `info` support to use floating window preview.

### CODE REVIEW

File: rplugin/python3/deoplete/sources/tabnine.py [hunk 1]
```diff
@@ -84,8 +84,12 @@ def gather_candidates(self, context):
                 candidate['abbr'] = word
             else:
                 candidate['word'] = word
-            if result.get('detailed'):
-                candidate['menu'] = result['detailed']
+            if result.get('detail'):
+                candidate['menu'] = result['detail']
+            if result.get('documentation'):
+                info = result['documentation'][:256]
+                info = re.sub(r'\n\n', r'\n', info)
```
Comment (tbodt) [inline rplugin/python3/deoplete/sources/tabnine.py]: Why remove double newlines?

Comment (Shougo) [reply rplugin/python3/deoplete/sources/tabnine.py]: Because, it consumes preview lines...
OK.  I will remove it.

Comment (tbodt) [reply rplugin/python3/deoplete/sources/tabnine.py]: Also why truncate at 256 characters? Both that and the removing of newlines seem like something that should be done by Deoplete so they can be configured by the user across all their sources.

Comment (Shougo) [reply rplugin/python3/deoplete/sources/tabnine.py]: OK.  I will do that.

**Comment by tbodt** [comment]
Thanks!

**Pull request merged**